diff -urN ../resources-original/icsm/geodesyml-1.0.0-SNAPSHOT/fieldMeasurement.xsd icsm/geodesyml-1.0.0-SNAPSHOT/fieldMeasurement.xsd
--- ../resources-original/icsm/geodesyml-1.0.0-SNAPSHOT/fieldMeasurement.xsd	2017-01-27 14:06:30.995000000 +1100
+++ icsm/geodesyml-1.0.0-SNAPSHOT/fieldMeasurement.xsd	2017-01-27 14:08:02.486788013 +1100
@@ -28,6 +28,7 @@
             <extension base="geo:DynamicFeatureType">
                 <sequence>
                     <element minOccurs="0" name="atSite" type="geo:SitePropertyType"/>
+<!-- TODO: this history element breaks JAXB
                     <element minOccurs="0" name="history">
                         <complexType>
                             <sequence>
@@ -35,6 +36,7 @@
                             </sequence>
                         </complexType>
                     </element>
+-->
                     <group ref="geo:RemarksGroup"/>
                 </sequence>
             </extension>
@@ -113,6 +115,7 @@
     </element>
     <!--  -->
     <complexType name="GNSSObservationQualityType">
+<!-- TODO: this history element breaks JAXB
         <complexContent>
             <extension base="geo:AbstractObservationQualityType">
                 <sequence>
@@ -126,6 +129,7 @@
                 </sequence>
             </extension>
         </complexContent>
+-->
     </complexType>
     <!--  -->
     <element name="GNSSObservationQualityInstance" substitutionGroup="geo:AbstractTimeSlice" type="geo:GNSSObservationQualityInstanceType"/>
diff -urN ../resources-original/icsm/geodesyml-1.0.0-SNAPSHOT/observationSystem.xsd icsm/geodesyml-1.0.0-SNAPSHOT/observationSystem.xsd
--- ../resources-original/icsm/geodesyml-1.0.0-SNAPSHOT/observationSystem.xsd	2017-01-27 14:06:30.993000000 +1100
+++ icsm/geodesyml-1.0.0-SNAPSHOT/observationSystem.xsd	2017-01-27 14:08:02.488787013 +1100
@@ -169,7 +169,9 @@
                             </restriction>
                         </simpleType>
                     </element>
+<!-- TODO: this location element breaks JAXB
                     <element minOccurs="0" name="Location" type="string"/>
+-->
                     <element minOccurs="0" name="LocalIDNumber" type="string"/>
                     <element minOccurs="0" name="MarkDescription" type="string"/>
                     <element minOccurs="0" name="GNSSReceiver" type="geo:InstrumentPropertyType"/>
@@ -397,6 +399,7 @@
                             <documentation>Estimation (adjustment) that defined position. As of GeodesyML 0.2 this can include any operation type. Retained (not deprecated) because this element can specify the default source for all elements in the history element</documentation>
                         </annotation>
                     </element>
+<!-- TODO: this history element breaks JAXB
                     <element name="history">
                         <complexType>
                             <sequence>
@@ -404,6 +407,7 @@
                             </sequence>
                         </complexType>
                     </element>
+-->
                     <group ref="geo:RemarksGroup"/>
                 </sequence>
                 <attributeGroup ref="geo:RequiredSRSReferenceGroup"/>
diff -urN ../resources-original/icsm/geodesyml-1.0.0-SNAPSHOT/referenceFrame.xsd icsm/geodesyml-1.0.0-SNAPSHOT/referenceFrame.xsd
--- ../resources-original/icsm/geodesyml-1.0.0-SNAPSHOT/referenceFrame.xsd	2017-01-27 14:06:30.994000000 +1100
+++ icsm/geodesyml-1.0.0-SNAPSHOT/referenceFrame.xsd	2017-01-27 14:08:02.490786013 +1100
@@ -179,6 +179,7 @@
                 <sequence>
                     <element ref="geo:Status" minOccurs="0"/>
                     <element name="source" type="geo:InterpolatedValueSourcePropertyType" minOccurs="0"/>
+<!-- TODO: this history element breaks JAXB
                     <element name="history">
                         <complexType>
                             <sequence>
@@ -186,6 +187,7 @@
                             </sequence>
                         </complexType>
                     </element>
+-->
                 </sequence>
             </extension>
         </complexContent>
